src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsPage.js

```diff
@@ -53,7 +58,7 @@ function MoneyRequestParticipantsPage({iou, selectedTab, route}) {
     const isScanRequest = MoneyRequestUtils.isScanRequest(selectedTab);
     const isSplitRequest = iou.id === CONST.IOU.MONEY_REQUEST_TYPE.SPLIT;
     const [headerTitle, setHeaderTitle] = useState();
-
+    const isEmptyWaypoint = _.isEmpty(lodashGet(transaction, 'comment.waypoints.waypoint0', {}));
```

Comment:
Is this the same as doing `lodashHas(transaction, 'comment.waypoints.waypoint0');`? If so, can you please simplify it in this way?

Another alternative that is maybe more accurate would be:

```
const hasWaypoints = _.size(lodashGet(transaction, 'comment.waypoints')) > 2;
```

Any transaction with 1 or less waypoints should be considered, invalid.

Reply:
@tgolen Thanks for your suggestion. I updated the logic for more accuracy based on the logic we used in DistanceReuquest https://github.com/Expensify/App/blob/5cf1904d08ea09c70db40151a14406dd5c1d956c/src/components/DistanceRequest.js#L293

Anyway, we need to resolve this problem https://github.com/Expensify/App/pull/25786#pullrequestreview-1647770453 first and then I can re-test this PR